great_expectations/expectations/core/expect_column_values_to_be_between.py

```diff
@@ -125,51 +125,7 @@ def validate_configuration(self, configuration: Optional[ExpectationConfiguratio
 
         # Setting up a configuration
         super().validate_configuration(configuration)
-        if configuration is None:
-            configuration = self.configuration
-
```

Comment:
`      if configuration is None:
            configuration = self.configuration

        min_val = None
        max_val = None
        # Setting these values if they are available
        if "min_value" in configuration.kwargs:
            min_val = configuration.kwargs["min_value"]

        if "max_value" in configuration.kwargs:
            max_val = configuration.kwargs["max_value"]
        try:
            # Ensuring Proper interval has been provided
            assert (
                min_val is not None or max_val is not None
            ), "min_value and max_value cannot both be None"
        except AssertionError as e:
            raise InvalidExpectationConfigurationError(str(e))
`